Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM node:8.1-alpine
+FROM node:8.2-alpine
 
-#RUN npm install -g create-react-app
-RUN yarn global add create-react-app
+# RUN yarn global add create-react-app
 
-WORKDIR /app
-
-COPY package.json /app
-
-RUN npm install --global
+WORKDIR /usr/src/app
+COPY package.json /usr/src/app
 RUN yarn install
 
-CMD ["yarn", "start"]
+COPY ./public /usr/src/app/public
+COPY ./src /usr/src/app/src
+RUN yarn build
 
 EXPOSE 3000
+CMD ["yarn", "start"]
+
+
+# FROM giantswarm/caddy:0.10.4-slim
```

Comment:
Ich würde gerne in `docker-compose` auf den ersten "build stage" verweisen, da das zum entwickeln gedacht ist. Aber es scheint, dass das nicht vergesehen ist. Vielleicht müssen wir also jeweils Dockerfile und -compose für `dev` und `prod` separat anlegen :disappointed: Aber würde das gerne erstmal so mergen und dann später erweitern. Ok?